.github/workflows/linter.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Lint Code Base
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: read
+      statuses: write
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Lint Code Base
+        uses: github/super-linter@v5
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          DEFAULT_BRANCH: master
+          VALIDATE_PYTHON_FLAKE8: true
+          PYTHON_FLAKE8_CONFIG_FILE: .flake8
```

Comment:
Guessing this means flake8 is the only linter being used?  That seems appropriate for a first pass, as it's more like what we had with stickler, though I think it'd be great to turn on pylint too if possible.

Reply:
Sure, I'll do that in a subsequent PR.